README.md

```diff
@@ -8,270 +8,16 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/hellofresh/klepto)](https://goreportcard.com/report/github.com/hellofresh/klepto)
 [![Go Doc](https://godoc.org/github.com/hellofresh/klepto?status.svg)](https://godoc.org/github.com/hellofresh/klepto)
 
-> Klepto is a tool for copying and anonymising data
+Klepto is a tool for copying and anonymising data
 
-Klepto is a tool that copies and anonymises data from other sources.
+- [Installation](docs/installation.md)
+- [Usage](docs/usage/commands.md)
+- [Configuration](docs/usage/config.md)
```

Comment:
what if we just point to the HelloDev page for the docs-related things?
this leaves for the repository readme to only have an introduction and contextualization

Reply:
I don't think would be a good choice, since this is a public repo not everyone that lands here will be able to go to HelloDev